tests/runtest.sh

```diff
@@ -370,6 +370,8 @@ function create_core_overlay {
         rm -f "$coreOverlayDir/System.Private.CoreLib.ni.dll"
     fi
     copy_test_native_bin_to_test_root
+
+    chmod +x $coreOverlayDir/corerun
```

Comment:
@wtgodbe @janvorli This seems like it shouldn't be necessary.  Do you happen know how/where this is done for other x64/Unix?  We appear to be missing something when building for ARM64.